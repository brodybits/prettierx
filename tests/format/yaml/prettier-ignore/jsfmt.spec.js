run_spec(__dirname, ["yaml"]);
// support --no-yaml-break-before-comment
run_spec(__dirname, ["yaml"], { yamlBreakBeforeComment: false });
