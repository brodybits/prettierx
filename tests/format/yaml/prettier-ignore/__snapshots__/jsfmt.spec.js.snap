// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`document.yml - {"yamlBreakBeforeComment":false} format 1`] = `
====================================options=====================================
parsers: ["yaml"]
printWidth: 80
yamlBreakBeforeComment: false
                                                                                | printWidth
=====================================input======================================
# prettier-ignore
---
aaaaa:
           bbbbb
...
aaaaa:
           bbbbb

=====================================output=====================================
# prettier-ignore
---
aaaaa:
           bbbbb
---
aaaaa: bbbbb

================================================================================
`;

exports[`document.yml format 1`] = `
====================================options=====================================
parsers: ["yaml"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
# prettier-ignore
---
aaaaa:
           bbbbb
...
aaaaa:
           bbbbb

=====================================output=====================================
# prettier-ignore
---
aaaaa:
           bbbbb
---
aaaaa: bbbbb

================================================================================
`;

exports[`leading-comment.yml - {"yamlBreakBeforeComment":false} format 1`] = `
====================================options=====================================
parsers: ["yaml"]
printWidth: 80
yamlBreakBeforeComment: false
                                                                                | printWidth
=====================================input======================================
abc:     123
# prettier-ignore
def:     456

=====================================output=====================================
abc: 123
# prettier-ignore
def:     456

================================================================================
`;

exports[`leading-comment.yml format 1`] = `
====================================options=====================================
parsers: ["yaml"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
abc:     123
# prettier-ignore
def:     456

=====================================output=====================================
abc: 123
# prettier-ignore
def:     456

================================================================================
`;

exports[`middle-comment.yml - {"yamlBreakBeforeComment":false} format 1`] = `
====================================options=====================================
parsers: ["yaml"]
printWidth: 80
yamlBreakBeforeComment: false
                                                                                | printWidth
=====================================input======================================
abc:  !!str #   hello
   123
# prettier-ignore
def:  !!str #   hello
   456

=====================================output=====================================
abc: !!str
  #   hello
  123
# prettier-ignore
def:  !!str #   hello
   456

================================================================================
`;

exports[`middle-comment.yml format 1`] = `
====================================options=====================================
parsers: ["yaml"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
abc:  !!str #   hello
   123
# prettier-ignore
def:  !!str #   hello
   456

=====================================output=====================================
abc: !!str
  #   hello
  123
# prettier-ignore
def:  !!str #   hello
   456

================================================================================
`;

exports[`trailing-comma.yml - {"yamlBreakBeforeComment":false} format 1`] = `
====================================options=====================================
parsers: ["yaml"]
printWidth: 80
yamlBreakBeforeComment: false
                                                                                | printWidth
=====================================input======================================
[
  # prettier-ignore
             {  sss  },       # 123
    {  qqqq },
  
  # prettier-ignore
          [ccc    ]       # 777
  , [ddd    ]
]

=====================================output=====================================
[
  # prettier-ignore
  {  sss  }, # 123
  { qqqq },

  # prettier-ignore
  [ccc    ], # 777
  [ddd],
]

================================================================================
`;

exports[`trailing-comma.yml format 1`] = `
====================================options=====================================
parsers: ["yaml"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
[
  # prettier-ignore
             {  sss  },       # 123
    {  qqqq },
  
  # prettier-ignore
          [ccc    ]       # 777
  , [ddd    ]
]

=====================================output=====================================
[
  # prettier-ignore
  {  sss  }, # 123
  { qqqq },

  # prettier-ignore
  [ccc    ], # 777
  [ddd],
]

================================================================================
`;

exports[`trailing-comment.yml - {"yamlBreakBeforeComment":false} format 1`] = `
====================================options=====================================
parsers: ["yaml"]
printWidth: 80
yamlBreakBeforeComment: false
                                                                                | printWidth
=====================================input======================================
abc:     123 # hello
# prettier-ignore
def:     456 # hello

=====================================output=====================================
abc: 123 # hello
# prettier-ignore
def:     456 # hello

================================================================================
`;

exports[`trailing-comment.yml format 1`] = `
====================================options=====================================
parsers: ["yaml"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
abc:     123 # hello
# prettier-ignore
def:     456 # hello

=====================================output=====================================
abc: 123 # hello
# prettier-ignore
def:     456 # hello

================================================================================
`;
